native/cocos/editor-support/spine/Json.cpp

```diff
@@ -378,15 +415,15 @@ const char *Json::parseNumber(Json *item, const char *num) {
         }
 
         while (*ptr >= '0' && *ptr <= '9') {
-            exponent = (exponent * 10.0) + (*ptr - '0');
+            exponent = (exponent * 10) + (*ptr - '0');
```

Comment:
*ptr is a value between `0` and `9`, exponent could only be an integer. So define it as uint32_t and use integer version of pow funciton (`MathUtil::ipow`) could make the package size smaller again.